packages/trace-viewer/src/ui/attachmentsTab.tsx

```diff
@@ -139,12 +149,16 @@ export const AttachmentsTab: React.FunctionComponent<{
     {attachments.size ? <div className='attachments-section'>Attachments</div> : undefined}
     {[...attachments.values()].map((a, i) => {
       return <div className='attachment-item' key={attachmentKey(a, i)}>
-        <ExpandableAttachment attachment={a} />
+        <ExpandableAttachment attachment={a} highlight={isActiveAttachment(a, selectedAction)} />
       </div>;
     })}
   </div>;
 };
 
+function isActiveAttachment(attachment: Attachment, activeAction: ActionTraceEventInContext | undefined): boolean {
+  return activeAction?.attachments?.some(a => a.name === attachment.name && a.path === attachment.path && a.sha1 === attachment.sha1) ?? false;
```

Comment:
- I guess if there are multiple attachments for the `activeAction`, you probably want to reveal just the first one?
- It seems to me that merely selecting a different action will reveal its attachments? Is that the UI we'd like? I'd prefer highlighting them instead, and only reveal upon an explicit button click.